site/docs/patterns/list-builder.mdx

```diff
@@ -0,0 +1,165 @@
+
+When moving items from the selected item list back to the pick list, position them according to the pick list's existing sort order. This makes items in large pick lists as quick and easy to find as possible.
+
+<ExampleContainer type="positive" customPillText="Best practices">
+- Change list labels based on your use case. Make sure to keep them clear and concise.
+- Always display a label for ADA compliance and to help users identify the purpose of the list, for example, a list of available/picked columns within a grid.
+- Show a count in brackets to help users keep track of the number of items in the list.
+- Incorporate more buttons into the button bar according to your use case. Examples include “move to top” and “move to bottom” functionalities.
+</ExampleContainer>
+
+## Multiple selection
+
+Users can move multiple items in a single action between the pick list and the selected item list using the list builder, allowing movement in both directions.
+
+  <Diagram
+    src="/img/patterns/list-builder/multi-select.png"
+    alt="Multiple selections with a list builder."
+    contentPosition="top"
+    border
+  />
+
+## Single selection
+
+Single selection allows users to move only one item at a time between the pick list and the selected item list.
+
+  <Diagram
+    src="/img/patterns/list-builder/single-select.png"
+    alt="Single selection with a list builder."
+    contentPosition="top"
+    border
+  />
+
+## Non-reorderable
+
+If users don’t need to reorder items in the new list, remove this feature.
+
+  <Diagram
+    src="/img/patterns/list-builder/non-reorderable.png"
+    alt="Single selection with a list builder."
+    contentPosition="top"
+    border
+  />
+
+## Empty state message
+
+Display a message to prompt or instruct the user when a list is empty. You can customize this message. Make sure that the text is concise, descriptive and clearly understood. Use the [content status pattern](/salt/patterns/content-status) to display empty state messages.
+
+  <Diagram
+    src="/img/patterns/list-builder/empty-state.png"
+    alt="An empty state message."
+    contentPosition="top"
+    border
+  />
+
+<ExampleContainer type="positive" customPillText="Best practices">
+Horizontally and vertically center the empty state message in the container.
+</ExampleContainer>
+
+## Stack layout
+
+Use a stack layout to compose the list builder vertically for interfaces with limited viewport width, such as mobile devices. 
+
+  <Diagram
+    src="/img/patterns/list-builder/stack-layout.png"
+    alt="A vertically aligned list builder."
+    contentPosition="top"
+    border
+  />
+
+<ExampleContainer type="positive" customPillText="Best practices">
+Use a minimum width of 8ems and ensure that the list builder spans the full width of the screen or its parent container.
```

Comment:
NIT: why recommend in an em value instead of px / % ?

Reply:
All min-widths are in ems